inst/examples/app/analytics/app.R

```diff
@@ -18,7 +18,7 @@ library(shiny.telemetry)
 
 # Default storage backend using LogFile
 data_storage <- DataStorageLogFile$new(
-  log_file_path = file.path(getwd(), "user_stats.txt")
+  log_file_path = file.path("../instrumentation/user_stats.txt")
```

Comment:
```suggestion
    log_file_path = log_file_path
```

Reply:
Did not think about this scenario. Nice catch!